### PR TITLE
Fix/more validation fixes

### PR DESCRIPTION
**Pull request by KetanReddy**
### Change Type (required)
- [x] `patch`
- [ ] `minor`
- [ ] `major`

## Release Notes
- XLR: Fix `Exclude` on Union Types
- XLR: Fix shallow clones causing unintended aftereffects 
- XLR SDK: Fix application of base Player Transforms to XLRs when loaded
- CLI: New Plugins to add Plugins to LSP and XLR SDK Transforms

### CODE REVIEW

File: xlr/sdk/src/utils.ts [hunk 1]
```diff
@@ -44,9 +56,23 @@ export function simpleTransformGenerator<
...
+                  ...token,
+                  constraints: token.constraints
+                    ? walker(token.constraints, capability)
+                    : undefined,
+                  default: token.default
+                    ? walker(token.default, capability)
+                    : undefined,
+                };
+              })
+            : undefined,
```
Comment (rafbcampos) [inline xlr/sdk/src/utils.ts:75]: Could you avoid adding `genericTokens: undefined`?
```suggestion
          ...(isGenericNamedType(node)
            ? {genericTokens: node.genericTokens.map((token) => {
                return {
                  ...token,
                  constraints: token.constraints
                    ? walker(token.constraints, capability)
                    : undefined,
                  default: token.default
                    ? walker(token.default, capability)
                    : undefined,
                };
              })}
            : {),
```

**Review by rafbcampos — APPROVED** [review]
LGTM

File: xlr/sdk/src/utils.ts (hunk 1 above)
Comment (KetanReddy) [reply xlr/sdk/src/utils.ts]: Good call, fixed.

**Pull request merged**